Testing/Tavisca.Testing/Tavisca.Training.Host/Host.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Tavisca.Testing.Model;
+
+
+namespace Tavisca.Training.Host
+{
+    class Host
+    {
+
+        static void Main(string[] args)
+        {
+            Assembly assembly = null;
+            try
+            {
+                 assembly = Assembly.LoadFrom(args[0]);
+
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+                Console.ReadKey();
+                return;
+            }
+            Console.WriteLine("Enter category:");
+            string category = Console.ReadLine();
+            if (string.IsNullOrWhiteSpace(category) == true)
+            {
+                Console.WriteLine("Invalid Category.");
+                Console.ReadKey();
+                return;
+            }
+            
+            foreach (Type type in assembly.GetTypes()) //test for each class in assembly
+            {
+                 if (type.IsClass)
+                 {
+                     Attribute[] ClassAttributes = Attribute.GetCustomAttributes(type);  // Reflection.
+                     foreach (Attribute attribute in ClassAttributes) //test for each attribute of that class   
+                     {
```

Comment:
What happens in the loop once the attribute matches the TestClassAttribute? It should break.